web_tool_script_1.R

```diff
@@ -23,6 +23,9 @@ set_portfolio_parameters(file_path = fs::path(par_file_path, paste0(portfolio_na
 
 set_project_parameters(file.path(working_location, "parameter_files",paste0("ProjectParameters_", project_code, ".yml")))
 
+data_location_ext <- sub("[0-9]{4}Q[0-9]", financial_timestamp, data_location_ext)
```

Comment:
```suggestion
data_location_ext <- sub("20[0-9]{2}Q[1-4]", financial_timestamp, data_location_ext)
```
I don't want to deal with false positives, and if we ever deal with data from before 2000, or (somehow) this is still running a millennium from now, we can change it then